dspace/config/modules/usage-statistics.cfg

```diff
@@ -40,3 +40,6 @@ usage-statistics.authorization.admin.workflow=true
 # Enable/disable if a matching for a bot should be case sensitive
 # Setting this value to true will increase cpu usage, but bots will be found more accurately
 #usage-statistics.bots.case-insensitive = false
+
+# Set to true if the statistics core is sharded into a core per year, defaults to false
+usage-statistics.shardedByYear = false
```

Comment:
Am I correct to assume that site's should set this to `true` if they are running `stats-util -s` to shard their index each year?  https://wiki.lyrasis.org/display/DSDOC6x/SOLR+Statistics+Maintenance#SOLRStatisticsMaintenance-SolrShardingByYear

If so, we should document that in this comment. by saying something like;
`If you are sharding your statistics index each year by running "dspace stats-util -s", you should set this to "true"`

Finally, as a general question, why did we default this to `false`?  It seems like in the current code, it's essentially defaulting to `true`...as `SolrLoggerServiceImpl.initSolrYearCores()` *always runs* by default and it checks to see if you have any year-based shards available.   So, is this fixing some bug you stumbled across with `initSolrYearCores()`?

Reply:
@tdonohue Quick update already:
We'll investigate this again since this was a while back that we created this, but re-reading the initial description, the problem here was that we noticed that the counts were being duplicated. So if the item count was triggered 3 times, the 'normal' core that was not sharded was being loaded twice, and the end result of the reported records was 6 counts.

Reply:
This config is to prevent the initSolrYearCores() from running if the core is not shared.
If this method ran the statisticYearCores became an ArrayList containing {solrUrlbase}/solr/statistics twice (resulting in all views being counted twice).
In initSolrYearCores the first one gets added because there is A solr statistics core (the only one), but then the seconds gets added at https://github.com/DSpace/DSpace/blob/bf7a8136189a3d93b9f9624b291daca69b12dae2/dspace-api/src/main/java/org/dspace/statistics/SolrLoggerServiceImpl.java#L1589, but there is only 1 statistics core. If the statistics core has not been sharded, there is no need to initialise the solrYearCores, so the config check made the most sense here.